### PR TITLE
Add a KB article about running multiple Redis queues

### PR DESCRIPTION
**Pull request by ghost**
Contains detailed information about how to run Redis queues for multiple sites on one server and how to configure this on Forge.

### CODE REVIEW

File: content/collections/kb/redis-queue-for-multiple-sites.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+title: Using a Redis queue for multiple sites on one server
+overview: You can run a Redis queue to optimize performance in the Control Panel when using Spock. With a Redis Queue you can also push other tasks, like generating assets, into the background. This article explains how you can make a Redis queue work for multiple Statamic sites on one server.
+kb_categories:
+  - Tips, Tricks, and How-Tos
+related_reading:
+  - 302be45f-bd9d-4a26-b836-d0f5c84b4773
+  - ec91d09e-c5a4-447e-b5a2-fd6ee2d7f89a
+---
```
Comment (jasonvarga) [inline content/collections/kb/redis-queue-for-multiple-sites.md:10]: Please an add ID here. Here's one I prepared earlier ✨ 

```
id: 03924f80-71cf-11e9-b475-0800200c9a66
```

Comment (studio1902) [reply content/collections/kb/redis-queue-for-multiple-sites.md:10]: There you go!

**Review by ghost — COMMENTED** [review]
Hope I did this right git wise.

**Comment by jasonvarga** [comment]
Looks good to me: https://docs.statamic.com/knowledge-base/redis-queue-for-multiple-sites

Thanks!

**Comment by studio1902** [comment]
Cool! Hope it helps some folks!

You’re welcome!

**Pull request merged**